docker_entrypoint.sh

```diff
@@ -1,3 +1,27 @@
 #!/bin/sh
 
-exec tini hello-world
+set -e
+
+ITCHYSATS_USER=itchysats
+ITCHYSATS_PASSWORD=$(yq e '.password' /root/start9/config.yaml)
+
+# Properties page for username and password
+echo "version: 2
+data:
+  ItchySats Username:
+    type: string
+    value: \"$ITCHYSATS_USER\"
+    description: The username for ItchySats
+    copyable: true
+    masked: false
+    qr: false
```

Comment:
This is outdated now. We don't have a username anymore.

Reply:
https://github.com/itchysats/start9-itchysats-wrapper/pull/1#issuecomment-1258096616